lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -325,19 +320,16 @@ private Optional<JourneyResponse> checkForProfileMatch(
             IpvSessionItem ipvSessionItem,
             ClientOAuthSessionItem clientOAuthSessionItem,
             AuditEventUser auditEventUser,
-            List<VcStoreItem> vcStoreItems,
+            List<VerifiableCredential> vcs,
             boolean areGpg45VcsCorrelated)
             throws ParseException, UnknownEvidenceTypeException, SqsException {
-        List<SignedJWT> credentials =
-                gpg45ProfileEvaluator.parseCredentials(
-                        userIdentityService.getIdentityCredentials(vcStoreItems));
+        var identityVcs = userIdentityService.getIdentityCredentials(vcs);
```

Comment:
getIdentityCredentials filters out TICF VC but we remove TICF VC earlier in the CheckExistingIdentityHandler handlerRequest method